src/systems/physics/Physics.cc

```diff
@@ -3259,7 +3259,11 @@ void PhysicsPrivate::UpdateCollisions(EntityComponentManager &_ecm)
         {
           msgs::Contact *contactMsg = contactsComp.add_contact();
           contactMsg->mutable_collision1()->set_id(_collEntity1);
+          contactMsg->mutable_collision1()->set_name(
+	    scopedName(_collEntity1, _ecm, ":", 1));
```

Comment:
My preference would be to not include the type of entity in the name. I would also not include the name of the world, since it should be clear from context, eg. the topic name. So instead of `world:contact_sensor:model:contact_model:link:link:collision:collision_sphere1`, we'd have `contact_model::link::collision_sphere1`.

Reply:
Done